### ISSUE TITLE
Access Requests not displayed in UI with the default 'access' role

### ISSUE THREAD

**Issue by philip-teleport** [gravitational/teleport#20135]
Expected behavior:

The Access Requests section should be displayed in the Web UI with the default `access` role.

Current behavior:

Now that [Issue 19054](https://github.com/gravitational/teleport/issues/19054) has been implemented, the Access Requests section is not displayed in the Web UI with the default `access` role.

In order to view the Access Requests section, the default `access` role needs to include the following permissions:

```
    - resources:
      - access_request
      verbs:
      - list
      - create
      - read
```

Bug details:
- Teleport version: 11.2.1
- Recreation steps: Deploy a new Teleport cluster, create a user with the default `access` role and access the web UI.
- Debug logs: N/A

**Comment by philip-teleport** [comment]
https://github.com/gravitational/teleport/issues/19054

**Comment by philip-teleport** [comment]
Note that there is also a possibility of a regression for existing deployments when they upgrade to `11.2.x` as `access_request` permissions did not need to explicitly set in roles in order for users to see the Access Requests section.

**Comment by zmb3** [comment]
FYI @mcbattirola

**Comment by mcbattirola** [comment]
Since `access` don't have permission to create access requests, even though the screen was available before https://github.com/gravitational/teleport/issues/19054, this role wouldn't be able to create an access request.

So what we really want is the default `access` role to have `create` on `access_request` by default, right?

**Comment by philip-teleport** [comment]
Yes, if the `access` role has `create` on `access_request` by default, then users will see the empty page with no selections available, as it was prior to 11.2.x:
![Screenshot 2023-01-12 at 19 03 09](https://user-images.githubusercontent.com/95687564/212157977-0c38e010-d951-4cec-8ea5-73ef6c5c3c8c.png)

**Comment by mcbattirola** [comment]
https://github.com/gravitational/teleport/issues/19054 is reverted and opened again. We can close this one.

I'll bring this up in the next UX/product sync so we decide the best way to handle this.

Thanks for reporting @philip-teleport.
